internal/apex/apex.go

```diff
@@ -543,13 +536,13 @@ func (ax *Apex) findLocalEndpointIp() (string, error) {
 
 // binaryChecks validate the required binaries are available
 func binaryChecks() error {
-	// Windows userspace binary
+	// Windows user space binary
 	if GetOS() == Windows.String() {
 		if !IsCommandAvailable(wgWinBinary) {
 			return fmt.Errorf("%s command not found, is wireguard installed?", wgWinBinary)
 		}
 	}
-	// Darwin wireguard-go userspace binary
+	// Darwin wireguard-go user space binary
```

Comment:
```suggestion
	// Darwin wireguard-go userspace binary
```

I would add userspace to the dictionary.